versioned_docs/version-2.0.0/06-deployments/05-general.md

```diff
@@ -4,7 +4,7 @@ You can host Serverpod anywhere, running Dart directly or through a Docker conta
 
 ## Required services
 
-Serverpod will not run without a link to a Postgres database with the correct tables added. Serverpod can also optionally use Redis. You enable Redis in your configuration files.
+Serverpod will not run without a link to a Postgres database with the correct tables added (unless you're running Serverpod mini). Serverpod can also optionally use Redis. You enable Redis in your configuration files.
```

Comment:
Is this understanding correct? (Based on https://medium.com/serverpod/getting-started-with-serverpod-mini-58c71ffefa54)

Reply:
Yes, this is correct! 👍